### DISCUSSION TITLE
How to get the appwrite.json when running Appwrite with Docker?

### DISCUSSION THREAD

**Discussion by devnm21** [appwrite/appwrite#3945] [Q&A] [answered]
I just got started with Appwrite and am really impressed with it!
I've started the Appwrite app with the following command as suggested: 
```
docker run -it --rm \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
    --entrypoint="install" \
    appwrite/appwrite:1.0.1

```
I understand that by using the appwrite-cli you can get the generated `appwrite.json` file for all the collections, functions that are created.
But, how do I get the same when using docker?

**Comment by stnguyen90** [comment, marked as answer]
The Appwrite CLI is independent of the Appwrite instance. When run, it will create an `appwrite.json` file in the current directory with data about your project and a `.appwrite` folder in your home directory that contains a `prefs.json` in your home directory.

When the install command is run, a `appwrite` folder is created in the current directory that contains the `docker-compose.yml` and `.env` files for Appwrite.

**Reply by devnm21** [reply]
Thanks for your answer. I can see an `appwrite` folder. 
So, is there a way to export the `.json` from an appwrite instance?
What I'm trying to achieve is to setup a development cycle with multiple contributors. For instance, developer A creates collection `Books` and dev B creates collection `Authors` through Appwrite instance, we wanna be able to review and merge it, then reload the instance with the new json - something like that perhaps.

What is a suggested way to build backend with Appwrite collaboratively? I really want to use Appwrite but this is the only thing that's stopping us.

**Reply by stnguyen90** [reply]
@devnm21, have you read through our docs for the [Appwrite CLI](https://appwrite.io/docs/command-line)?
